Core.ApplicationServices/SystemUsage/Migration/ItSystemUsageMigrationService.cs

```diff
@@ -252,57 +229,30 @@ public bool CanExecuteMigration()
             return _authorizationContext.HasPermission(new SystemUsageMigrationPermission());
         }
 
-        private bool DeleteExhibits(ItSystemUsageMigration migration)
+        private bool DeleteUsedByRelationInterfaces(ItSystemUsageMigration migration)
         {
-            var exhibitsToBeDeleted = migration.AffectedContracts.SelectMany(x => x.ExhibitUsagesToBeDeleted);
-            foreach (var itInterfaceExhibitUsage in exhibitsToBeDeleted)
+            var relationWithInterfaceToBeDeleted = migration.AffectedRelations
+                .Where(x => x.ItInterfaceToBeDeleted && x.Relation.RelationInterfaceId != null)
```

Comment:
Du skal kun søge på interfaceToBeDeleted, for hvis den er sat, så har du vel allerede tjekket at der er en interface binding? ... ellers så bør du da